### PR TITLE
Hiro/bugfix

### PR DESCRIPTION
**Pull request by yasuhiro112358**
[Hiro/bugfix]
- fix layout.app for whole view
- fix the view profiles.show
- fix the view favorites.index
- fix the view posts.show
- fix the view browsing-history.index

- remove unnecessary nav and footer
- fix ProfileController and routes
- fix view of profile.edit
- fix modal window for destroy method
- fix design
- fix default profile

### CODE REVIEW

File: app/Http/Controllers/PostController.php [hunk 1]
```diff
@@ -281,28 +299,13 @@ public function update(Request $request, $id)
         return redirect()->route('posts.show', $id);
     }
 
-
     public function destroy($id)
```
Comment (kredo-school) [inline app/Http/Controllers/PostController.php:302]: can you use the same as approach as this?
 $user = $this->user->findOrFail($user_id);
        $user->delete();

File: public/js/posts/show.js [hunk 2]
```diff
@@ -2,10 +2,13 @@
 
 {
     // ==== Env ====
-    const token = document.querySelector('meta[name="csrf-token"]').getAttribute('content');
     const hostUrl = 'http://localhost:8000';
```
Comment (kredo-school) [inline public/js/posts/show.js:5]: make hostUrl dynamic for cases when we want to deploy your project in heroku or other servers

File: app/Http/Controllers/PostController.php (hunk 1 above)
Comment (yasuhiro112358) [reply app/Http/Controllers/PostController.php:302]: I change the approach to delete post.

File: public/js/posts/show.js (hunk 2 above)
Comment (yasuhiro112358) [reply public/js/posts/show.js]: I make it dynamic.

**Pull request merged**